src/pages/versions.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import Layout from '@theme/Layout';
+
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import versions from '../../versions.json';
+
+function Version() {
+  const context = useDocusaurusContext();
+  const {siteConfig = {}} = context;
+  const latestVersion = versions[0];
+  const pastVersions = versions.filter(version => version !== latestVersion);
+  const pastVersionsReadTheDocs = [
+    '4.0.5',
+    '3.4.2',
+    '3.3.0',
+    '3.2.1',
+    '3.1.0',
+    '3.0.0',
+    '2.0.0'
+  ]
+  const repoUrl = `https://github.com/${siteConfig.organizationName}/${siteConfig.projectName}`;
+  return (
+    <Layout
+      permalink="/versions"
+      description="Versions page listing all documented site versions">
+      <div className="container margin-vert--xl">
```

Comment:
@BMartinos Could you please make this page match the other custom pages to keep the experience consistant.
It just needs the header element witht he image and title

Reply:
great thanks Matt, will implement :+1:

Reply:
Done in e5a737a and fdb068a